OpenSim/Common/ComponentConnector.h

```diff
@@ -344,36 +381,99 @@ class Connector : public AbstractConnector {
     }
 
     SimTK_DOWNCAST(Connector, AbstractConnector);
+    
+    /** For use in python/java/MATLAB bindings. */
+    // This method exists for consistency with Object's safeDownCast.
+    static Connector<T>* safeDownCast(AbstractConnector* base) {
+        return dynamic_cast<Connector<T>*>(base);
+    }
+    
+protected:
+    /** Create a Connector that can only connect to Object of type T with 
+    specified name and stage at which it should be connected. Only Component
+    should ever construct this class.
```

Comment:
should --> can?
As defined only a derived `Connector` or `Component` *can* invoke this constructor.